src/components/AppDrawerAuth.js

```diff
@@ -1,10 +1,26 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Link } from "react-router-dom";
 import List from "@material-ui/core/List";
 import ListItem from "@material-ui/core/ListItem";
 import ListItemText from "@material-ui/core/ListItemText";
 import Progress from "./Progress.js";
+import { withStyles } from "@material-ui/core/styles";
+
+const styles = theme => ({
+  noPadding: {
+    paddingTop: 0,
+    paddingBottom: 0,
+  },
+  smallerTextGroup: {
+    paddingTop: 20,
+    paddingBottom: 20
+  },
+  smallerLink: {
+    '&:hover': {
+      cursor: "pointer"
```

Comment:
The `ListItemText` component doesn't make links obvious. I had to replicate the behaviour of a link to not impact the UX. Feels a bit hacky though.